sql_generators/experiment_monitoring/templates/experiment_enrollment_aggregates_v1/query.sql

```diff
@@ -25,6 +25,21 @@ WITH
         event_category = 'normandy' AND
         submission_date = @submission_date
     ),
+  {% elif app_dataset == "monitor_cirrus" %}
```

Comment:
```suggestion
  {% elif "_cirrus" in app_dataset %}
```

All of this will apply to new Cirrus applications as well, right?

Reply:
yes